solver/app/tokens_internal_test.go

```diff
@@ -127,25 +126,3 @@ func TestTokenResponse(t *testing.T) {
 
 	tutil.RequireGoldenJSON(t, resp, tutil.WithFilename("TestTokens/tokens_response.json"))
 }
-
-func TestMaxSpendMinThreshold(t *testing.T) {
-	t.Parallel()
-
-	for _, token := range tokens.All() {
-		if !IsSupportedToken(token) {
-			continue
-		}
-
-		bounds, ok := tokenSpendBounds[token.Asset][token.ChainClass]
-		if !ok {
-			continue
-		}
-
-		thresh, ok := eoa.GetSolverNetThreshold(eoa.RoleSolver, netconf.Mainnet, token.ChainID, token.Asset)
-		if !ok {
-			continue
-		}
-
-		tutil.RequireGTE(t, thresh.MinBalance(), bounds.MaxSpend, "solver min balance must be greater than max spend: token=%s, min_bal=%s, max_spend=%s", token.Asset, thresh.MinBalance(), bounds.MaxSpend)
-	}
-}
```

Comment:
Removing this test. Min balances are now lower. Max spend should be around target balances. But we'll need to start differentiating max spend definitions between chains, which we don't right now.

Plus we have InsufficientInventory rejections, which is good.